src/js/actions/tools.js

```diff
@@ -753,6 +754,20 @@ define(function (require, exports) {
                 });
             }
         }
+
+        if (vectorMaskMode) {
+            var layerKind = "";
```

Comment:
Suggestion: just make the event equal to `"mask-mode-" + layer.kind`. We can figure out the mapping from number to name later, and then we'll catch all the layer kinds. (E.g., we're missing at least smart objects below.) Bonus: it's easier. Also, the tool can change during mask mode... But, I think we can either ignore that OR just not track the particular tool. Both seem fine.